src/gen/lib/AST_grammar_conv.ml

```diff
@@ -44,8 +44,8 @@ let translate find_alias (x : Tree_sitter_t.rule_body) =
     | PREC_RIGHT (_opt_prio, x) -> translate x
     | FIELD (_name, x) -> translate x (* TODO not sure about ignoring this *)
     | ALIAS alias -> translate_alias find_alias alias
-    | IMMEDIATE_TOKEN x -> translate x (* TODO check what this is *)
-    | TOKEN x -> translate x
+    | IMMEDIATE_TOKEN _body -> String "immediate token"
+    | TOKEN _body -> String "token"
```

Comment:
I think this is incorrect. I'll investigate.

Reply:
This made me discover a few other things I misunderstood about aliases and some bugs we may run into. I tried to simplify things but it's not getting that simpler and it would require a rewrite.

I'm now leaning toward eliminating all support for aliases. It would simplify our program a lot and eliminate bugs. We just have to eliminate aliases from the grammar.js files, which shouldn't be a problem. I was hoping to preserve them to give us more insightful type names in the AST but it's looking like a lost cause.

Reply:
I'm all in favor of simplifying the code and our job by adjusting the grammar.js, especially because we will at some point anyway modify those grammars (e.g., to add ellipsis).

Reply:
Pending confirmation that will come in today, I think we'll be able to remove support for aliases and hidden rules completely.